python/pyarrow/tests/test_pandas.py

```diff
@@ -4891,20 +4886,6 @@ def make_df_with_timestamps():
     return df
 
 
-@pytest.mark.parquet
-@pytest.mark.filterwarnings("ignore:Parquet format '2.0':FutureWarning")
-def test_timestamp_as_object_parquet(tempdir):
-    # Timestamps can be stored as Parquet and reloaded into Pandas with no loss
-    # of information if the timestamp_as_object option is True.
-    df = make_df_with_timestamps()
-    table = pa.Table.from_pandas(df)
-    filename = tempdir / "timestamps_from_pandas.parquet"
-    pq.write_table(table, filename, version="2.0")
```

Comment:
same question as above, doesn't this work with parquet 2.6? What is the problem?

Reply:
https://github.com/apache/arrow/pull/45849#discussion_r2012270921